src/app/room.ts

```diff
@@ -0,0 +1,4 @@
+export interface Room {
+  roomId: number;
```

Comment:
if the id is of the entity, just name it as id.

Reply:
Changed